block_designs.tex

```diff
@@ -263,20 +401,26 @@ \section{Cyklické blokové plány a diferenčné množiny}
 
 \end{theorem_hard}
 
-\begin{remark}
-
-Existujú aj ďalšie triedy diferenčných množín, napríklad bikvadratické rezíduá alebo tzv. \emph{twin prime power difference set}.
-
-\end{remark}
-\begin{toreview}
+\begin{theorem_hard}{(Typ B dif. množín --- bikvadratické rezíduá)}\\
+	Nech $F := GF(q)$ je konečné pole, kde $q = p^l$ a $p = 4x^2 + 1$ pre nepárne $x$. 
+	Nech $r$ je generátor grupy $F = (GF(q), +)$. 
+	Potom množina bikvadratických rezíduí grupy $F$ $BQR(F) := \set{TODO}$
```

Comment:
This TODO is intended?

Reply:
Yes, I didn't manage to write a correct and pretty definition of this set.